configtx/channel.go

```diff
@@ -142,3 +142,9 @@ func (c *ChannelGroup) RemoveCapability(capability string) error {
 
 	return nil
 }
+
+// RemoveLegacyOrdererAddresses removes legacy orderer addresses config key and value
```

Comment:
I think we should (or at least could) expand on the comment here to clarify what we mean by "legacy" orderer addresses. Just something talking about how previously orderer endpoints were specified globally (to use the term I've seen in the logs) and now are specified at the orderer organization level.

Reply:
I agree with Will, maybe mention in fabric 1.4, top level orderer addresses were migrated to the org level orderer endpoints. Fabric still supports both but prefers the org level value if they exist. The addition of the function is to create an update path for consumers wishing to remove the deprecated top level orderer addresses. It might be worth adding an example test for this as well where we start with a fabric 1.0 config that has top level orderer addresses and then we show how to add the org level orderer endpoints and subsequently remove the top level orderer addresses. Maybe call it Example_MigrateLegacyOrdererAddresses or something like that.

Reply:
I expanded on the comment but I don't think the example is necessary.